src/Service/ProductBundleRegistryInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Created by solutionDrive GmbH.
+ *
+ * @copyright 2018 solutionDrive GmbH
+ */
+
+namespace SolutionDrive\SyliusProductBundlesPlugin\Service;
+
+use SolutionDrive\SyliusProductBundlesPlugin\Entity\ProductBundleInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+
+interface ProductBundleRegistryInterface
+{
+    public function isProductBundle(ProductInterface $product): bool;
+
+    /**
+     * @param ProductInterface $product
+     *
+     * @return ProductBundleInterface|object
+     */
+    public function getProductBundleForProduct(ProductInterface $product): object;
```

Comment:
Why 'object' as return type and not directly 'ProductBundleInterface'?

Reply:
See commit history: if I use the correct interface, phpstan complains about wrong return type, because the Repository is annotated with "object" as return type.

Reply:
ok